internal/capability/operand_cleanup.go

```diff
@@ -2,20 +2,19 @@ package capability
 
 import (
 	"context"
-	"fmt"
-
 	"github.com/opdev/opcap/internal/operator"
 
 	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-)
+	"go.uber.org/zap"
+)	
 
 // OperandCleanup removes the operand from the OCP cluster in the ca.namespace
 func (ca *CapAudit) OperandCleanUp() error {
-	logger.Debugw("cleaningUp operand for operator", "package", ca.Subscription.Package, "channel", ca.Subscription.Channel, "installmode", ca.Subscription.InstallModeType)
+	zap.S().Debug("CleaningUp operand for operator", "package", ca.Subscription.Package, "channel", ca.Subscription.Channel, "installmode", ca.Subscription.InstallModeType)
```

Comment:
This was cleaned up (ha!) in @acmenezes #188 patch. Should be "cleaning up"

Reply:
Thanks, after the rebase got fixed.